expo/scheduler/templates/scheduler/tag_templates/volunteer_schedule.tmpl

```diff
@@ -0,0 +1,9 @@
+{% if schedule %}
```

Comment:
Is there any reason to call this a "volunteer schedule"?  It's really just a schedule...  reusable for all sorts of great things.